.travis.yml

```diff
@@ -63,10 +63,16 @@ jobs:
           script: TAG=$TRAVIS_TAG TARGET=release ./scripts/ci_docker.sh
           on:
             tags: true
+
+    - stage: "Github Pages"
```

Comment:
This should actually be part of the same stage as the docker deploy, otherwise the build will take longer because it will wait for the docker stage to complete before moving to the GH pages deploy stage. Take a look at the testing jobs above, they all run in the same stage but as parallel jobs.

Reply:
good to know